src/api/types/generated/fragments/stop-places.ts

```diff
@@ -3,4 +3,5 @@ export type StopPlaceFragment = {
   name: string;
   latitude?: number;
   longitude?: number;
+  isFree?: boolean;
```

Comment:
These fragments shouldn't be manually edited, as they are one-to-one with the generated fragments in the BFF.

Reply:
I made a fix for this, good observation :)